lib/auth_browser.dart

```diff
@@ -54,7 +54,7 @@ Client clientViaApiKey(String apiKey, {Client baseClient}) {
 /// if one was given.
 Future<BrowserOAuth2Flow> createImplicitBrowserFlow(
     ClientId clientId, List<String> scopes,
-    {Client baseClient}) {
+    {Client? baseClient}) {
   if (baseClient == null) {
```

Comment:
We can avoid the `!` below if we introduce a new var

```dart
var refCountedClient = RefCountedClient(baseClient ?? BrowserClient(), initialRefCount: baseClient == null ? 1 : 2);
```

Reply:
Done.